soc/arm64/arm/fvp_aemv8r/Kconfig.defconfig.series

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2021 Arm Limited (or its affiliates). All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_FVP_AEMV8R
+
+config SOC_SERIES
+	default "fvp_aemv8r"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 100000000
+
+config NUM_IRQS
+	default 128
+
+config 2ND_LVL_ISR_TBL_OFFSET
```

Comment:
I'm not sure this is needed.  At one point the main interrupt controller was hacked into the arm irq using this define for the v7 version of cortex-r but that has been fixed.

Reply:
No need anymore. Thanks for your reminder, I'll remove it